app/services/payment_requests/payments/stripe_service.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
+
+module PaymentRequests
+  module Payments
+    class StripeService < BaseService
+      include Customers::PaymentProviderFinder
+
+      PENDING_STATUSES = %w[processing requires_capture requires_action requires_confirmation requires_payment_method]
+        .freeze
+      SUCCESS_STATUSES = %w[succeeded].freeze
+      FAILED_STATUSES = %w[canceled].freeze
+
+      def initialize(payable = nil)
+        @payable = payable
+
+        super(nil)
```

Comment:
```suggestion
        super
```

Reply:
that would call initialize in BaseService that expects current_user as argument and will pass the payable object instead. This requires `super(nil)`.